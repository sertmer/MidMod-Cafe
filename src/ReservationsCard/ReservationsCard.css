article {
  background-color: white;
  border: 1px solid teal;
  border-radius: 5px;
  padding: 10px;
}

button {
  background-color: teal;
  color: white;
}