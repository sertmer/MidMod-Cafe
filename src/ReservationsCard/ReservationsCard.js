import React from 'react'
import './ReservationsCard.css'

const ReservationsCard = ({id, name, date, time, number, removeReservation }) => {
  console.log(id)
  return (
    <article>
      <h2>{name}</h2>
      <p>{date}</p>
      <p>{time}</p>
      <p>Number of Guests: {number}</p>
      <button
      onClick={() => removeReservation(id)}>cancel</button>
    </article>
  )
}

export default ReservationsCard