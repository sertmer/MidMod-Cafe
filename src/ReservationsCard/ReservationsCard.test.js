import React from 'react';
import { shallow } from 'enzyme';
import ReservationsCard from './ReservationsCard';

describe('ReservationsCard', () => {
  let wrapper
  let mockReservation
  let mockRemoveReservation
  beforeEach(() => {
    mockReservation = [{
      "id": 1,
      "name": "Christie",
      "date": "12/29",
      "time": "7:00",
      "number": 12
  }]

    mockRemoveReservation = jest.fn
    wrapper = shallow(<ReservationsCard 
      {...mockReservation}  
      removeReservation={mockRemoveReservation}
    />)
  })
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})