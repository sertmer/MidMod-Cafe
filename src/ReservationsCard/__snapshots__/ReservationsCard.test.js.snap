// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationsCard should match the snapshot 1`] = `
<article>
  <h2 />
  <p />
  <p />
  <p>
    Number of Guests: 
  </p>
  <button
    onClick={[Function]}
  >
    cancel
  </button>
</article>
`;
